tools_docker/README.md

```diff
@@ -159,9 +158,7 @@ Example with basic instance of Tracim (local usage with webdav and caldav) with
 Note: with this new docker, all tracimcli and alembic command should be runned as
 user www-data, example:
 
-```bash
-su www-data -s /bin/bash -c "tracimcli dev parameters value -f -d -c /etc/tracim/development.ini"
-```
+docker exec -it -u www-data -w /etc/tracim {CONTAINER ID} tracimcli dev parameters value -f -d
```

Comment:
```suggestion
```bash
docker exec -it -u www-data -w /etc/tracim {CONTAINER ID} tracimcli dev parameters value -f -d
```
```